web/src/components/tasks/business-formation/review/section/ReviewLineItem.tsx

```diff
@@ -12,6 +12,7 @@ interface Props {
   dataTestId?: string;
   marginOverride?: string;
   formatter?: (value: string) => string;
+  noColenAfterLabel?: boolean;
```

Comment:
dust - why is this being added, it feels weird to not have a Colon for 1 item on the review page. I noticed there are no designs in the ticket, I'd check with the design team about this.

Reply:
We explicitly asked content about it and this is what they wanted.

Reply:
I'd check with design. One of their principles is consistency. I'd be surprised if they agree with Content. 

Or at a Min, add design to review the ticket and write a comment to design letting them know the formatting is different.

Reply:
[pebble] in the meantime, while it's still part of the code, we should change `noColen...` to `noColon...`